### PR TITLE
update output - expose API version

### PR DESCRIPTION
**Pull request by klokane**
Related to https://github.com/apiaryio/fury-adapter-swagger/pull/207

### CODE REVIEW

File: fixtures/features/swagger/body-schema-example.yaml [hunk 1]
```diff
@@ -19,7 +19,6 @@ paths:
         200:
           description: response description
           schema:
-            type: object
```
Comment (kylef) [inline fixtures/features/swagger/body-schema-example.yaml]: Why was this removed?

File: fixtures/features/api-elements/body-schema-example.sourcemap.json [hunk 2]
```diff
@@ -491,41 +520,7 @@
                               ]
                             }
                           },
-                          "content": "{\"type\":\"object\",\"examples\":[{\"id\":123,\"name\":\"Resource 1\"}]}"
```
Comment (kylef) [inline fixtures/features/api-elements/body-schema-example.sourcemap.json]: I don't think this should be removed.

File: fixtures/features/api-elements/body-schema-example.json [hunk 3]
```diff
@@ -268,41 +272,7 @@
                               "content": "application/schema+json"
                             }
                           },
-                          "content": "{\"type\":\"object\",\"examples\":[{\"id\":123,\"name\":\"Resource 1\"}]}"
```
Comment (kylef) [inline fixtures/features/api-elements/body-schema-example.json]: I don't think this should be removed.

**Review by kylef — CHANGES_REQUESTED** [review]

File: fixtures/features/swagger/body-schema-example.yaml (hunk 1 above)
Comment (klokane) [reply fixtures/features/swagger/body-schema-example.yaml]: Sorry some mistake, will revert

**Comment by klokane** [comment]
Updated

**Review by kylef — APPROVED** [review]

**Review by kylef — CHANGES_REQUESTED** [review]
The commit messages for these commits don't follow our commit message format.

The commits should look something like:

feat: Expose API Version in parse result
chore: Release 2.16.0

@klokane I can push these changes and we can merge this. Although I don't want to force push to your branch without permission.

**Comment by klokane** [comment]
Feel free to update. It is team work :)

**Review by kylef — APPROVED** [review]
